Makefile

```diff
@@ -14,6 +14,8 @@
 # Tasks provide shortcuts to common operations that occur frequently during
 # development. This includes running configured linters and executing unit tests
 
+SHELL = /bin/bash
```

Comment:
would this cause problems if someone is not using bash? maybe set it in the workflow only?

Reply:
Done.  Switched it to the github workflow.